packages/schemas/tables/sign_in_experiences.sql

```diff
@@ -7,6 +7,7 @@ create table sign_in_experiences (
   id varchar(21) not null,
   color jsonb /* @use Color */ not null,
   branding jsonb /* @use Branding */ not null,
+  hide_logto_branding boolean not null default false,
```

Comment:
We have Redis cache applied. Will legacy data break the zod guard if this field is missing?

Reply:
The `BaseCache` will do type check and ignore invalid data